client/my-sites/stats/activity-log-banner/error-banner.jsx

```diff
@@ -76,10 +76,8 @@ class ErrorBanner extends PureComponent {
 					details: translate( 'We came across a problem while trying to restore your site.' ),
 				}
 			: {
-					title: translate( 'Problem creating a file' ),
-					details: translate(
-						'There was a problem building your backup into a downloadable file.'
-					),
+					title: translate( 'Problem preparing your backup' ),
```

Comment:
this seems like we get right back to the confusion which brought us here. I know the details explains more, but maybe we should be careful with "problem with your backup"